### PR TITLE
[js_upload] Fixing missing extension index

### PR DESCRIPTION
**Pull request by nupplaphil**
Fixing https://github.com/friendica/friendica/issues/6338#issuecomment-472373099

### CODE REVIEW

File: js_upload/js_upload.php [hunk 1]
```diff
@@ -333,7 +334,10 @@ function handleUpload(){
         $pathinfo = pathinfo($this->file->getName());
         $filename = $pathinfo['filename'];
 
-        $ext = $pathinfo['extension'];
+        if (!isset($pathinfo['extension'])) {
+        	Logger::warning('extension isn\'t set.', ['filename' => $filename]);
+		}
```
Comment (MrPetovan) [inline js_upload/js_upload.php]: Code standard: Mixed indentation.

Comment (nupplaphil) [reply js_upload/js_upload.php]: Fixed the indentation for the whole file ;)

File: js_upload/js_upload.php [hunk 2]
```diff
@@ -106,22 +106,22 @@ function createUploader() {
 
 			if (typeof acl != "undefined"){
 				uploader.setParams( {
-					newalbum      : newalbum,
-					album         : album,
+					newalbum	  : newalbum,
+					album		 : album,
```
Comment (MrPetovan) [inline js_upload/js_upload.php]: Please revert these alignment change that must remain spaces.

Comment (nupplaphil) [reply js_upload/js_upload.php]: I already reverted it with https://github.com/friendica/friendica-addons/pull/827/commits/e0815309d0812d5b4dc5fa8ba7b9195712ee272b

**Pull request merged**